packages/groqd-playground/src/Playground.tsx

```diff
@@ -1,9 +1,105 @@
 import * as React from "react";
+import { useClient } from "sanity";
+import { z } from "zod";
+import * as q from "groqd";
+import { BaseQuery } from "groqd/src/baseQuery";
+
+export default function GroqdPlayground() {
+  const [query, setQuery] = React.useState<BaseQuery<any>>(q.q(""));
+  const [response, setResponse] = React.useState("");
+  const client = useClient({ apiVersion: "v2021-10-21" });
+
+  const runQuery = React.useMemo(
```

Comment:
does client change or can you make this a ref?

Reply:
I think `client` _can_ change. Right now, memo vs ref probably doesn't make a huge difference – so I can ref this.